lib/ansible/cli/scripts/ansible_connection_cli_stub.py

```diff
@@ -148,6 +148,22 @@ def run(self):
                     resp = self.srv.handle_request(data)
                     signal.alarm(0)
 
+                    # This should be handled elsewhere, but if this is the last task, nothing will
```

Comment:
This can be moved into a method and also invoked from connect_timeout, command_timeout and handler

Reply:
Yeah, I've done some of that locally. Rather than individually in each handler, though, I replaced the existing message dumping in `shutdown`, which should get called unconditionally in every case.